resources/views/global/_image_cta_block.antlers.html

```diff
@@ -0,0 +1,33 @@
+<section class="bg-split-white-concrete">
+  <div class="xl:container xl:py-100 relative">
+    {{ svg src="graphic-pattern-plus" class="absolute top-45 xl:block hidden" }}
```

Comment:
could save writing an extra class here

```suggestion
    {{ svg src="graphic-pattern-plus" class="absolute top-45 xl-max:hidden" }}
```

Reply:
*this comment applies sitewide*